src/tag/TagItemCollection.h

```diff
@@ -38,12 +38,15 @@ namespace UKControllerPlugin {
                     UKControllerPlugin::Euroscope::EuroscopePluginLoopbackInterface & pluginCore
                 ) const;
 
-                // The tag item string to use if the tag item is too long.
-                const std::string invalidTagItemText = "INVALID";
-
                 // The tag item string to use if the tag item requested is invalid
                 const std::string errorTagItemText = "ERROR";
 
+                // The tag item text is too long
+                const std::string invalidItemText = "INVALID";
+
+                // Max length we can have on TAG items, 15 characters + 1 null terminator
+                const size_t maxItemLength = 16;

```

Comment:
Minor thing but since we are using size() and not length(), do you want to change this name to maxItemSize?